tox.ini

```diff
@@ -47,7 +48,7 @@ commands=
   ansible-playbook -vv -i {changedir}/hosts {toxinidir}/{env:PURGE_PLAYBOOK:purge-cluster.yml} --extra-vars "\
       ireallymeanit=yes \
       remove_packages=yes \
-      ceph_stable_release={env:CEPH_STABLE_RELEASE:luminous} \
+      ceph_stable_release={env:CEPH_STABLE_RELEASE:dev} \
```

Comment:
I think we can keep 'nautilus' as default value (so the next stable-release is ready)